template/java8/model/src/test/java/ResponseTest.java

```diff
@@ -49,4 +51,18 @@
       r.setStatusCode(404);
       assertEquals(404, r.getStatusCode());
     }
+
+    @Test public void testResponseSetBodyWorksAsBefore() {
```

Comment:
> WorksAsBefore

^ this won't mean anything after the merge. How about _WithString and WithBytes ?

Reply:
Yes, I agree. I will change this method name.